### PR TITLE
Update popover explainer with recent changes

### PR DESCRIPTION
**Pull request by mfreed7**
As part of the WHATWG spec landing process, several changes have recently been made to the Popover API:
 1. The invoking attributes were changed significantly. The earlier three attributes were changed to two new attributes with different behaviors.
 2. The animations behavior was not specified at all, and instead, new CSSWG issues are in progress to re-allow animations.

This PR updates the explainer accordingly.

### CODE REVIEW

File: research/src/pages/components/popover.research.explainer.md [hunk 1]
```diff
@@ -300,61 +300,56 @@ Be mindful when using other stylesheets that you haven't authored. These may set
...
+- https://github.com/w3c/csswg-drafts/issues/8389 (`inert`ing things that animate to `display:none`)
+
+Assuming all five of the above features land in specs and browsers in roughly the form they're currently being discussed, then to animate a popover's show/hide transitions you would use:
 
 ```css
 [popover] {
-  opacity: 0;
-  transition: opacity 0.5s;
+  --transition-time: 0.5s;
+  transition: display var(--transition-time), overlay-behavior var(--transition-time), opacity var(--transition-time);
```
Comment (mfreed7) [inline research/src/pages/components/popover.research.explainer.md:318]: @flackr would you mind reviewing my examples and descriptions in this section, to make sure I'm not lying?

**Comment by brechtDR** [comment]
+1 on the attribute changes in the explainer, they seem correct. I should take a bit of a deep-dive for the CSS animations and the issues open there, so no opinion on that for the moment.

**Review by andrico1234 — APPROVED** [review]

**Comment by mfreed7** [comment]
Thanks!

**Comment by jh3y** [comment]
So we've gone with `display` and `overlay-behavior` as the required transitional properties?

**Comment by jh3y** [comment]
p.s Looks great 👏 Not sure if it's been added but something around `:open`, `:closed` might be worth mentioning. Imagine most developers are purely going to want to see "What's the simplest way to fade the thing in and out?"

**Comment by mfreed7** [comment]
> p.s Looks great 👏 Not sure if it's been added but something around `:open`, `:closed` might be worth mentioning. Imagine most developers are purely going to want to see "What's the simplest way to fade the thing in and out?"

Thanks! I included a small snippet with what that looks like, but of course it could change as all these things are standardized.

**Pull request merged**